stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/IdentifierSpec.kt

```diff
@@ -8,6 +8,7 @@ import kotlinx.serialization.Serializable
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 enum class ApiParameterDestination {
     Params,
+    Extras,
```

Comment:
We should redo this structure, as `ApiParameterDestination` explicitly refers to the Stripe API. The `Extras` are client-only.